mnist-cnn/label_errors_mnist_train_cnn.ipynb

```diff
@@ -2404,7 +2432,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "Python 3.9.13 64-bit ('3.9.13')",
```

Comment:
You may still have some weird settings in your Jupyter?  Normally this should say:
`Python 3 (ipykernel)`

Reply:
Hm seems like it might be a VSCode issue. Reverted the display names to `Python 3 (ipykernel)`